samples/azure-quantum/check-ghz/CheckGHZ.ipynb

```diff
@@ -0,0 +1,191 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Integrated Hybrid: Blending classical and quantum computing\n",
+        "In this sample, you will discover how to blend classical and quantum instructions in the same program, all fully processed by the quantum computing back-end.\n",
+        "You can learn more how to program with Integrated hybrid at https://aka.ms/AQ/Hybrid/Docs\n",
+        "\n",
+        "This program will examine a 3 qubit Greenberger–Horne–Zeilinger (GHZ) state and counts the number of times it sees the entanglement fail out of 10 attempts. Unlike in perfect, noiseless simulations, entanglement failures can occur in real-world runs, which are subject to the effects of noise.\n",
```

Comment:
```suggestion
        "This program will examine a 3 qubit Greenberger–Horne–Zeilinger (GHZ) state and count the number of times it sees the entanglement fail out of 10 attempts. Unlike in perfect, noiseless simulations, entanglement failures can occur in real-world runs, which are subject to the effects of noise.\n",
```

Reply:
The sample is run against a simulator with simulated noise (for quantinuum). Is this statement about "real-world runs" supposed to be about hardware?

Reply:
@cesarzc could you comment?